_data/events_gdcr2019/durban.json

```diff
@@ -0,0 +1,28 @@
+{
+    "title": "GDCR2019 in Durban @ Kaleidocode",
+    "url": "https://www.meetup.com/coderetreat/events/265958153/",
+    "moderators": [
+      {
+        "name": "Brendon Page",
+        "url": "https://brendonpage.dev/"
+      }
+    ],
+    "sponsors": [
+      {
+        "name": "Kaleidocode",
+        "url": "http://www.kaleidocode.co.za/"
+      }
+    ],
+    "date": {
+      "start": "2019-11-16T09:00:00.000+02:00",
+      "end": "2019-11-16T17:00:00.000+02:00"
+    },
+    "location": {
+      "city": "Durban",
+      "country": "South Africa",
+      "coordinates": {
+        "latitude": 45.8358475,
+        "longitude": 9.0262319
+      }
```

Comment:
Those coordinates point to a place in italy 😬

![image](https://user-images.githubusercontent.com/418117/67621789-9d61a500-f813-11e9-85b1-d88ce1dcf9e9.png)